code/modules/supermatter/supermatter.dm

```diff
@@ -84,12 +84,12 @@ var/global/list/supermatter_final_thoughts = list(
 /obj/machinery/power/supermatter
 	name = "supermatter crystal"
 	desc = "A strangely translucent and iridescent crystal. <span class='danger'>You get headaches just from looking at it.</span>"
-	icon = 'icons/obj/engine.dmi'
-	icon_state = "darkmatter"
+	icon = 'icons/obj/supermatter.dmi'
+	icon_state = "supermatter"
 	density = 1
 	anchored = 0
 	light_range = 4
-	layer = ABOVE_OBJ_LAYER
+	layer = ABOVE_HUMAN_LAYER
```

Comment:
I really don't like this. I get it's a hack because the icon is too big, but nothing else in the game does this. Would much rather the icon was smaller.